STRUCTURE.md

```diff
@@ -0,0 +1,105 @@
+A [conversion funnel](https://www.google.com/search?q=conversion+funnel&tbm=isch) plans the site visitor stage goals.
+
+1. Strangers: Discovery, Awareness, Attract, Brand
+    * Facebook, YouTube, Snapchat, Instagram, Twitter, Press
+    * Measure impressions/reach
+
+2. Visitors: Landing Page, Realization of Need, Interest/Desire, Engage leads, Familiarity
+    * Assert the unmet needs the "one key fan" users face, and how the products/services addresse their needs
+    * Survey their needs/expectations
+    * Calls To Action
+    * Measure active daily users
+
+3. Leads: Consideration, Try Outs, Action Flows
+    * Sugarizer inherently allow immediate try out
+    * download and boot SOAS
+    * tell your friends to try it out too
+
+4. Users: Conversion/Sales
+    * Create account our central Sugarizer server and (1) bookmark or (2) install as an app 
+    * Sugar Desktop or Sugarizer Server installed and used daily
+      * either buying hardware with it pre-installed, or 
+      * self-install on a laptop, desktop or personal server (eg raspberry pi)
+
+5. Participants: Retention/Loyalty
+    * Schedule for 12+ months of usage (curricular scale)    
+    * Actively participate in the community
+      * discuss on mailing list
+      * translate
+      * file bugs, make pull requests
+    * Advocate on behalf of sugar labs
+      * contribute to social media
+      * do public speaking
+
+## Primary/Secondary Elements
+
+A coherent structure of pages. 
+Eventually this will be expressed in an ["information architecture" site map diagram](https://www.google.com/search?q=information+architecture+web+design&tbm=isch).
+
+The elements on the 1st level of the list form the 'primary navigation' that typically appears on a toolbar at the top of the page. 
+The elements on the 2nd/3rd levels of the list are the 'secondary navigation' and may appear in a drop down menu on the primary toolbar, or as links inside the landing page each primary element, etc.
+
+home
+* What do middle school kids need? How Sugar addresses those needs.
+* "A successful technology creates problems that only it can solve." - Alan Kay
+* Story telling: one great case study
+* Calls To Action
+
+about
+* "Learning To Change The World" full text liberated
+* case studies
+   * a table listing each user community ("deployment") and a person in each community who Sugar Labs can talk with? (Starting with http://wiki.laptop.org/go/Manufacturing_data)
+* detailed information about the Sugar Labs organization
+   * Explanation of Conservancy
+   * SLOB Minutes
+   * Current SLOB members
+   * SLOB historical membership data
+   * Finance data
+   * Current Members data
+   * historical membership data
+
+try now
+* http://server.sugarizer.org with guiders
+
+download
+* SOAS
+* Sugar Handbook (User Manual, tutorial videos)
+* Activities Library (http://activities.sugarlabs.org)
+* eBook Library (including interactive OERs, curricula)
+
+buy
+* offer sugar sticks (branded usb keys) 
```

Comment:
Through the "Recycle USB" program, we have hundreds of USB keys available for SoAS. We could develop a program for shipping USBs on demand.